app/views/users/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render partial: "shared/main_menu", locals: { items: true  } %>
+<div style="text-align: center;">
+  <a href="/register" class="button">Create New User</a>
```

Comment:
Either change `href="/register"` to `href=<%= register_user_path %>` or delete the `as: register_user` from the routes file. The `as: register_user` is useless extra code unless you add `register_user_path` somewhere.

Reply:
Good point - made this change